docs/apm-input-settings.asciidoc

```diff
@@ -499,6 +503,8 @@ Trace name
 `trace.name`
 
 | (string) The trace name for events to match a policy.
+A match occurs when the configured `trace.name` matches the `transaction.name` of the root transaction of a trace.
+A root transaction is any transaction with a `parent.id`.
```

Comment:
```suggestion
A root transaction is any transaction without a `parent.id`.
```
I think root transaction should be a transaction without a parent ID.

Reply:
Ah, thank you!